app.py

```diff
@@ -60,12 +59,6 @@ def initialize_database(autoinitialize=True):
     for logger in logging.getLogger().handlers:
         app.log.info("Logs are going to %r", logger)
 
-    # Workaround for a "Resource temporarily unavailable" error when
```

Comment:
The socket needs to be set before the database is initialized. This has been moved into the `run` method.